pkg/storage/fs/posix/tree/inotifywatcher_default.go

```diff
@@ -0,0 +1,22 @@
+//go:build !linux
+
+// Copyright 2025 OpenCloud GmbH <mail@opencloud.eu>
+// SPDX-License-Identifier: Apache-2.0
+
+package tree
+
+import (
+	"github.com/opencloud-eu/reva/v2/pkg/errtypes"
+	"github.com/rs/zerolog"
+)
+
+// DummyWatcher is a dummy watcher that does nothing
+type DummyWatcher struct{}
```

Comment:
NoopWatcher?

Reply:
NullWatcher? It's not to be used at all after all.